css_files/rules.css

```diff
@@ -0,0 +1,13 @@
+
+.rule{
+    font-size: 1.6rem;
+    margin-bottom: var(--sm-space);
+}
+
+.btn-rules{
+    font-size: 1.6rem;
+    display: block;
+    text-align: center;
+    width: 60%;
+    margin: 2.4rem auto auto auto;
```

Comment:
You can simply use `margin-top: 2.4rem;` it would work the same

Reply:
Okay